src/serac/physics/tests/thermal_statics_patch.cpp

```diff
@@ -176,7 +176,7 @@ double solution_error(const ExactSolution& exact_temperature, PatchBoundaryCondi
   HeatTransfer<p, dim> thermal(nonlinear_opts, heat_transfer::direct_linear_options, heat_transfer::default_static_options, "thermal", mesh_tag);
 
   heat_transfer::LinearIsotropicConductor mat(1.0,1.0,1.0);
-  thermal.setMaterial(std::move(mat));
```

Comment:
@white238, from what I can tell this is the most likely culprit of the segfault you're seeing. `mat` is being moved from, then used again after the move when it is likely is some invalid state.